Doc/library/os.rst

```diff
@@ -3535,9 +3536,9 @@ written in Python, such as a mail server's external command delivery program.
 
    If *mode* is :const:`P_NOWAIT`, this function returns the process id of the new
    process; if *mode* is :const:`P_WAIT`, returns the process's exit code if it
-   exits normally, or ``-signal``, where *signal* is the signal that killed the
-   process.  On Windows, the process id will actually be the process handle, so can
-   be used with the :func:`waitpid` function.
+   exits normally, or ``-signal`` on non-VxWorks platform, where *signal* is the
+   signal that killed the process. On Windows, the process id will actually be the
+   process handle, so can be used with the :func:`waitpid` function.
```

Comment:
I would prefer to see a separated paragraph to describe the behavior on VxWorks, rather than writing "non-VxWorks paltforms".